spec/rack_test_spec.rb

```diff
@@ -12,6 +12,7 @@ module TestSessions
   screenshot
   frames
   windows
+  active_element
```

Comment:
This would only make a difference if you did `requires: [:active_element]` instead of :js on the tests